packages/ui/.storybook/preview.ts

```diff
@@ -8,7 +8,8 @@ const preview: Preview = {
 		backgrounds: { disable: true },
 		controls: {
 			matchers: {
-				color: /(background|color)$/i,
+				// matches "background" and "color" but not "solidBackground" as that is a boolean arg
+				color: /(?<!solid)(background|color)$/i,
```

Comment:
this silents a console warning that the controls addon was logging in a story that uses `solidBackground`. The console warning was more noticeable in the Vitest tests because it would show up in the output.